src/web/modules/gateway_accounts/api_keys.njk

```diff
@@ -1,6 +1,7 @@
 {% from "components/button/macro.njk" import govukButton %}
 {% from "components/table/macro.njk" import govukTable %}
-{% extends "layout/layout.njk" %} 
+{% from "common/jsonDetails.njk" import json %}
```

Comment:
Maybe it would be better not to confuse json with jsonDetails (applies to all usages):
```suggestion
{% from "common/jsonDetails.njk" import jsonDetails %}
```

Reply:
Will rename the file name to match, json feels cleanest to me as details doesn't have any meaning